lib/cext/ruby.h

```diff
@@ -38,6 +38,16 @@ extern "C" {
 
 #include <truffle.h>
 
+
+// Used in st.h
+#define RUBY_SYMBOL_EXPORT_BEGIN
+#define RUBY_SYMBOL_EXPORT_END
+#define SIZEOF_LONG 8
+#define SIZEOF_LONG_LONG 8
+#define SIZEOF_VOIDP 8
```

Comment:
These were the values on my system. I wasn't certain if there is a good constant value for these or they need to be generated by system. Used in `st.h`